ride_confirmation_bp.py

```diff
@@ -0,0 +1,54 @@
+from flask import Blueprint, render_template, request, redirect, url_for, flash, abort
+from flask_login import current_user
+import db_functions
+import email_functions
+
+blueprint = Blueprint('ride_confirmation_bp', __name__)
+
+
+@blueprint.route('/potvrzenijizdy/<chosen>')
+def show_ride_confirmation(chosen):
+	chosen = db_functions.choose_carpool(chosen)
+	user = current_user
+	if user.is_authenticated:
+		return render_template('potvrzeni_jizdy.html', values=chosen)
+	else:
+		flash('Abys mohl nastoupit, musíš se nejdřív přihlásit.', "danger")
+		return render_template('prihlaseni.html')
+
+
+@blueprint.route('/potvrzenijizdy/', methods=['POST'])
+def want_to_board_on():
```

Comment:
Since this function is called when user submits a form created in the `show_ride_confirmation` function, how about we call this `confirm_ride`?